src/esp_littlefs.c

```diff
@@ -899,13 +899,17 @@ static int esp_littlefs_flags_conv(int m) {
 
 static void esp_littlefs_take_efs_lock(void) {
     if( _efs_lock == NULL ){
+#ifdef ESP32  // ESP8266 only has one core, no need for SMP protection
```

Comment:
I think checking the following is more robust/proper:

```
#if portNUM_PROCESSORS > 1
    ...
#endif
```

Reply:
Oh very good point!